package.json

```diff
@@ -40,5 +41,8 @@
     "vscode-uri": "^1.0.6",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.9"
+  },
+  "dependencies": {
+    "tslint": "^5.20.0"
```

Comment:
tslint should be dev dependencies

Reply:
Right, I forgotten add a parameter to a command.